frontend/src/pages/signup.tsx

```diff
@@ -22,6 +22,8 @@ export default function SignUpPage() {
   const [values, setValues] = useState<stateInterface>({} as stateInterface);
```

Comment:
use a react state for the loading state:
`const [loading, setLoading] = useState(false);`